lib/features/family/features/parent_giving_flow/presentation/pages/organization_list_family_page.dart

```diff
@@ -0,0 +1,194 @@
+  const OrganizationListFamilyPage({
+    super.key,
+  });
+
+  @override
+  State<OrganizationListFamilyPage> createState() =>
+      _OrganizationListFamilyPageState();
+}
+
+class _OrganizationListFamilyPageState
+    extends State<OrganizationListFamilyPage> {
+  final TextEditingController controller = TextEditingController();
+
+  @override
+  void dispose() {
+    controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final locals = context.l10n;
+    final bloc = context.read<OrganisationBloc>();
+    return Scaffold(
+      appBar: const TopAppBar(
+        leading: GivtBackButtonFlat(),
+        title: 'Give',
+      ),
+      body: BlocConsumer<OrganisationBloc, OrganisationState>(
+        listener: (context, state) {
+          if (state.status == OrganisationStatus.error) {
+            ScaffoldMessenger.of(context).showSnackBar(
+              SnackBar(
+                content: Text(locals.somethingWentWrong),
+              ),
+            );
+          }
+        },
+        builder: (context, state) {
+          return Column(
+            children: [
+              const SizedBox(
+                height: 16,
+              ),
+              _buildFilterType(bloc, locals),
+              Padding(
+                padding: const EdgeInsets.symmetric(
+                  horizontal: 24,
+                  vertical: 16,
+                ),
+                child: FamilySearchField(
+                  autocorrect: false,
+                  controller: controller,
+                  onChanged: (value) => context
+                      .read<OrganisationBloc>()
+                      .add(OrganisationFilterQueryChanged(value)),
+                ),
+              ),
+              if (state.status == OrganisationStatus.filtered)
+                Expanded(
+                  child: ListView.separated(
+                    separatorBuilder: (_, index) => const Divider(
+                      height: 0.1,
+                    ),
+                    shrinkWrap: true,
+                    itemCount: state.filteredOrganisations.length,
+                    itemBuilder: (context, index) {
+                      return _buildListTile(
+                        type: state.filteredOrganisations[index].type,
+                        title: state.filteredOrganisations[index].orgName,
+                        onTap: () {
+                          context.read<OrganisationBloc>().add(
+                                OrganisationSelectionChanged(
+                                  state.filteredOrganisations[index].nameSpace,
+                                ),
+                              );
+                          _navigateToGivingScreen(state, context);
+                        },
+                      );
+                    },
+                  ),
+                )
+              else
+                const Center(
+                  child: CircularProgressIndicator(),
```

Comment:
Should we change this indicator to new style as well? I think its CustomCircularProgressIndicator

Reply:
Missed that, thanks!